morf-h2/pom.xml

```diff
@@ -42,5 +42,9 @@
       <artifactId>log4j</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>

```

Comment:
Generally, we don't introduce compile-time dependencies on JDBC drivers in dialect modules.

Do we think H2 is different?  It is in Maven Central, it's open source - there's no real reason to avoid it.

We need other runtime stuff too, like the inspector, so I'm happy if we all are.